includes/polling/ports.inc.php

```diff
@@ -445,15 +445,12 @@
             $port['update']['poll_period'] = $polled_period;
         }
 
-        // Copy ifHC[In|Out]Octets values to non-HC if they exist
-        if ($this_port['ifHCInOctets'] > 0 && is_numeric($this_port['ifHCInOctets']) && $this_port['ifHCOutOctets'] > 0 && is_numeric($this_port['ifHCOutOctets'])) {
-            echo 'HC ';
-            $this_port['ifInOctets']  = $this_port['ifHCInOctets'];
+        // use HC values if they are available
+        if (isset($this_port['ifHCInOctets'])) {
```

Comment:
Won't this always be set?  Shouldn't it be `if (!isset($this_port['ifInOctets')) {`